sql/dd_table_share.cc

```diff
@@ -759,6 +759,11 @@ static bool fill_share_from_dd(THD *thd, TABLE_SHARE *share,
     table_options->get("encryption_key_id", &share->encryption_key_id);
   }
 
+  if (table_options->exists("explicit_encryption")) {
+    table_options->get("explicit_encryption", &bool_opt);
+    share->explicit_encryption = bool_opt;
```

Comment:
Why not `table_options->get("explicit_encryption", &share->explicit_encryption);`

Reply:
fixed :)